src/OpenTelemetry.AutoInstrumentation/Logging/OtelLogging.cs

```diff
@@ -61,7 +62,13 @@ private static IOtelLogger CreateLogger(string suffix)
             {
                 var fileName = GetLogFileName(suffix);
                 var logPath = Path.Combine(logDirectory, fileName);
-                sink = new FileSink(logPath);
+                sink = new RollingFileSink(
+                    path: logPath,
+                    fileSizeLimitBytes: FileSizeLimitBytes,
+                    retainedFileCountLimit: 10,
+                    rollingInterval: RollingInterval.Day,
```

Comment:
Then we will have max 10 day history?

Reply:
Correct, do you think we should increase that?